src/main/java/io/jenkins/plugins/audit/CustomHTMLLayout.java

```diff
@@ -0,0 +1,412 @@
+
+        private final String size;
+
+        private FontSize(final String size) {
+            this.size = size;
+        }
+
+        public String getFontSize() {
+            return size;
+        }
+
+        public static FontSize getFontSize(final String size) {
+            for (final FontSize fontSize : values()) {
+                if (fontSize.size.equals(size)) {
+                    return fontSize;
+                }
+            }
+            return SMALL;
+        }
+
+        public FontSize larger() {
+            return this.ordinal() < XXLARGE.ordinal() ? FontSize.values()[this.ordinal() + 1] : this;
+        }
+    }
+
+    private CustomHTMLLayout(final boolean locationInfo, final String title, final String contentType, final Charset charset,
+            final String font, final String fontSize, final String headerSize) {
+        super(charset);
+        this.locationInfo = locationInfo;
+        this.title = title;
+        this.contentType = addCharsetToContentType(contentType);
+        this.font = font;
+        this.fontSize = fontSize;
+        this.headerSize = headerSize;
+    }
+
+    /**
+     * For testing purposes.
+     */
+    public String getTitle() {
+        return title;
+    }
+
+    /**
+     * For testing purposes.
+     */
+    public boolean isLocationInfo() {
+        return locationInfo;
+    }
+
+    @Override
+    public boolean requiresLocation() {
+        return locationInfo;
+    }
+
+    private String addCharsetToContentType(final String contentType) {
+        if (contentType == null) {
+            return DEFAULT_CONTENT_TYPE + "; charset=" + getCharset();
+        }
+        return contentType.contains("charset") ? contentType : contentType + "; charset=" + getCharset();
+    }
+
+    /**
+     * Formats as a String.
+     *
+     * @param event The Logging Event.
+     * @return A String containing the LogEvent as HTML.
+     */
+    @Override
+    public String toSerializable(final LogEvent event) {
+        final StringBuilder sbuf = getStringBuilder();
+
+        sbuf.append(Strings.LINE_SEPARATOR).append("<tr>").append(Strings.LINE_SEPARATOR);
+
+        sbuf.append("<td>");
+        sbuf.append(event.getTimeMillis() - jvmStartTime);
+        sbuf.append("</td>").append(Strings.LINE_SEPARATOR);
+
+        final String escapedThread = Transform.escapeHtmlTags(event.getThreadName());
+        sbuf.append("<td title=\"").append(escapedThread).append(" thread\">");
+        sbuf.append(escapedThread);
+        sbuf.append("</td>").append(Strings.LINE_SEPARATOR);
+
+        sbuf.append("<td title=\"Level\">");
+        if (event.getLevel().equals(Level.DEBUG)) {
+            sbuf.append("<font color=\"#339933\">");
+            sbuf.append(Transform.escapeHtmlTags(String.valueOf(event.getLevel())));
+            sbuf.append("</font>");
+        } else if (event.getLevel().isMoreSpecificThan(Level.WARN)) {
+            sbuf.append("<font color=\"#993300\"><strong>");
```

Comment:
If you define CSS classes for these font tags, you can simplify the `<td>` to use the class and not have inline styles here. Not necessary to merge, but it would make this layout a little easier to read in my opinion.

Reply:
yes, good option to keep all css classes at one places. updated it.